From d50443b19c6b2f879450fbe73420978beb43e58f Mon Sep 17 00:00:00 2001
From: Al Muthanna Athamina <almuthanna@chromium.org>
Date: Wed, 10 Nov 2021 17:54:43 +0100
Subject: [PATCH] [infra] Clean up mb_config after splitting the rest of the
 ports console builders

Bug: chromium:1136844
Change-Id: I1c9be9ff38114f548b5f40462d96968dbf1565ab
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3272580
Commit-Queue: Michael Achenbach <machenbach@chromium.org>
Reviewed-by: Michael Achenbach <machenbach@chromium.org>
Reviewed-by: Liviu Rau <liviurau@chromium.org>
Cr-Commit-Position: refs/heads/main@{#77829}
---
 infra/mb/mb_config.pyl | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/infra/mb/mb_config.pyl b/infra/mb/mb_config.pyl
index f92485df36f..aaca329e51f 100644
--- a/infra/mb/mb_config.pyl
+++ b/infra/mb/mb_config.pyl
@@ -180,23 +180,23 @@
       'V8 Arm - builder': 'release_arm',
       'V8 Arm - debug builder': 'debug_arm',
       'V8 Android Arm - builder': 'release_android_arm',
-      'V8 Linux - arm - sim': 'release_simulate_arm',
-      'V8 Linux - arm - sim - debug': 'debug_simulate_arm',
+      'V8 Linux - arm - sim - builder': 'release_simulate_arm',
+      'V8 Linux - arm - sim - debug builder': 'debug_simulate_arm',
       'V8 Linux - arm - sim - lite - builder': 'release_simulate_arm_lite',
       'V8 Linux - arm - sim - lite - debug builder': 'debug_simulate_arm_lite',
       # Arm64.
       'V8 Android Arm64 - builder': 'release_android_arm64',
       'V8 Android Arm64 - debug builder': 'debug_android_arm64',
       'V8 Arm64 - builder': 'release_arm64_hard_float',
-      'V8 Linux - arm64 - sim': 'release_simulate_arm64',
-      'V8 Linux - arm64 - sim - debug': 'debug_simulate_arm64',
-      'V8 Linux - arm64 - sim - gc stress': 'debug_simulate_arm64',
+      'V8 Linux - arm64 - sim - builder': 'release_simulate_arm64',
+      'V8 Linux - arm64 - sim - debug builder': 'debug_simulate_arm64',
+      'V8 Linux - arm64 - sim - gc stress - builder': 'debug_simulate_arm64',
       # Mips.
       'V8 Linux - mipsel - sim - builder': 'release_simulate_mipsel',
       'V8 Linux - mips64el - sim - builder': 'release_simulate_mips64el',
       # IBM.
-      'V8 Linux - ppc64 - sim': 'release_simulate_ppc64',
-      'V8 Linux - s390x - sim': 'release_simulate_s390x',
+      'V8 Linux - ppc64 - sim - builder': 'release_simulate_ppc64',
+      'V8 Linux - s390x - sim - builder': 'release_simulate_s390x',
       # RISC-V
       'V8 Linux - riscv64 - sim - builder': 'release_simulate_riscv64',
       # Loongson
-- 
2.35.1

